lib/usd/translators/mayaReferenceUpdater.cpp

```diff
@@ -177,6 +176,19 @@ bool PxrUsdTranslators_MayaReferenceUpdater::editAsMaya()
     return true;
 }
 
+bool callEditRouter(
```

Comment:
Will be useful if we change the edit router API to return a success/failure value or if we allow explicitly to throw exception on error, we will be able to catch the exception here and convert it to a success/failure value.